### PR TITLE
Add hint text for Google Sheets URL field

### PR DESCRIPTION
**Pull request by shouryalala**
Hi @yanokwa, apologies for the late update. Exams have had the best of me.

Okay so after going through a few cases this is what i gathered
1. The submission url in the form always takes precedence over the GoogleSheetsURL field
2. The 'uploadOneSubmission' method in the GoogleSheetsAbstractUploader class specifies that only urls' of the form "../docs.google.com/spreadsheets/d/..." will be accepted. It makes the verification of the url more easier but i feel that there is a trade off. If a person is present at site and needs to send a form to a separate spreadsheet, he/she might need to use a shortened url(goo.gl/, bit.do/, etc ) as it really isnt possible to remember a spreadsheet url. But this can be taken up later i guess. 
3. The code otherwise is very robust. Any Google spreadsheet can be used to upload form data as long as the first row of the sheet is empty.
4. If there is no url provided in either the form or the settings field, it gives an invalid sheet id message.

The quick addition i made is that i added an appropriate hint text in the settings GoogleSheetsUrl field:- 
"http://docs.google.com/spreadsheets/d/sheet_id"

Note: There are no changes in the readme. I had edited it in the master branch by accident while looking into a different issue

### CODE REVIEW

File: collect_app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -314,6 +314,7 @@
 <string name="aggregate_preferences">ODK Aggregate settings</string>
 <string name="other_preferences">Other platform settings</string>
 <string name="google_preferences">Google Sheets settings</string>
+<string name="google_sheets_url_hint"><font size="13">http://docs.google.com/spreadsheets/d/sheet_id</font></string>
```
Comment (yanokwa) [inline collect_app/src/main/res/values/strings.xml]: We want to warn people that this field is only for the case where they downloaded a form with no submission URL. 

`Do not use! This is for the unlikely case when a blank form has an empty Google Sheets URL.`

Comment (yanokwa) [reply collect_app/src/main/res/values/strings.xml]: Also, there shouldn't be a font size. Is there a reason you added it?

File: README.md [hunk 2]
```diff
@@ -81,3 +81,4 @@ Current and previous production builds can be found on the [ODK website](https:/
 
 #### Android Studio Error: `SDK location not found. Define location with sdk.dir in the local.properties file or with an ANDROID_HOME environment variable.`
 When cloning the project from Android Studio, click "No" when prompted to open the `build.gradle` file and then open project.
+
```
Comment (shobhitagarwal1612) [inline README.md]: Please remove this blank new line

**Comment by shobhitagarwal1612** [comment]
@shouryalala I think a regex for the required sheets url should be written. Then match the inputted data with that regex. If it doesn't match, then display error message that the url is not well defined

**Comment by lognaturel** [comment]
@shobhitagarwal1612 Agreed but we can do that as a separate issue. Here we just wanted to make sure it was clear what that strange setting for sheets URL actually means.

**Comment by lognaturel** [comment]
@shouryalala I was going to quickly get this ready for merging but now I'm not totally clear on what the user should see! Thanks for clarifying what this field does, I think we will finalize the decision on what to do with it for the next revision. I'm leaving this open for now pending a final decision @yanokwa

**Pull request merged**